docs/fundamentals/code-analysis/style-rules/ide0110.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "IDE0110: Discard can be removed"
+description: "Learn about code analysis rule IDE0110: Discard can be removed"
+ms.date: 11/02/2020
+ms.topic: reference
+f1_keywords:
+- IDE0110
+helpviewer_keywords:
+- IDE0110
+author: CyrusNajmabadi
+ms.author: CyrusNajmabadi
+dev_langs:
+- CSharp
+---
+# Discard can be removed (IDE0110)
+
+|Property|Value|
+|-|-|
+| **Rule ID** | IDE0080 |
+| **Title** | Discard can be removed |
+| **Category** | Style |
+| **Subcategory** | Unnecessary code rules |
+| **Applicable languages** | C# |
+
+## Overview
+
+This rule flags unnecessary [discard patterns](https://github.com/dotnet/csharplang/blob/master/proposals/csharp-8.0/patterns.md#discard-pattern), when this discard pattern is used in a context where it has no effect. 
```

Comment:
Can the hyperlink point to a documentation page for discards instead of the specification? Tag @gewarren

Reply:
Two comments here. First, I've added a note to #20584 that the discard pattern should be covered. (It wasn't called out in the issue, and our current language reference doesn't cover it explicitly).

Second, for now, let's update the link to point to the pattern spec that's published on docs. This helps me in two ways. First, as we get to the updated language reference, I'll find the link and replace it. Second, when we intend to link to the proposals, I'll find and update those links when the proposal text moves to the official standard. (Bonus nit: this suggestion removes the whitespace that's causing the markdown lint warning.

```suggestion
This rule flags unnecessary [discard patterns](../../_csharplang/proposals/csharp-8.0/patterns.md#discard-pattern), when this discard pattern is used in a context where it has no effect.
```